### ISSUE TITLE
Ad Astra Incompatibility

### ISSUE THREAD

**Issue by FooterManDev** [MelanX/SkyblockBuilder#142]
### Minecraft version

1.19.2

### LibX version

4.2.8

### Skyblock Builder version

4.2.14

### Forge version

43.2.21

### The latest.log file

https://gist.github.com/FooterMan15/2a00ec388f167760678744ae7738b393

### Issue description

When playing on a Skyblock world with [Ad Astra](https://github.com/terrarium-earth/Ad-Astra), and launching off in a Rocket, then coming back to Earth(Overworld) generates actual Minecraft terrain... in a Skyblock world, for 12 chunks in all directions.

Ad Astra devs say it's not their fault, so I came to ask if when something tries to generate terrain it can instead be blocked and the Rocket can fall to the void?

### Steps to reproduce

Install: [ Skyblock Builder, LibX ], [ Ad Astra, Botarium, Resourceful Lib, Resourceful Config (per Ad Astra required dependencies) ].
Load a Skyblock world.
Make a Rocket and launch.
In the Solar System GUI, select Earth.

### Other information

Ad Astra: 1.12.6
Resourceful Lib: 1.1.24
Resourceful Config: 1.0.20
Botarium: 1.8.2

Modpack: 0.1.5 - https://github.com/TheGangModpacks/Fun-in-the-Sky

**Comment by MelanX** [comment]
I tried to reproduce it but I'm not able to. First, I installed the pack and tried it. You disabled earth, that's good, but even after enabling, there was no normal world. I travelled a bit and couldn't find any weird chunks. 
After this, I thought I'll try the exact versions you provided, but still nothing. Sadly, I don't know how I could help you with this. And yes, I saw the image on Ad Astra issue tracker here: https://github.com/terrarium-earth/Ad-Astra/issues/263

**Comment by FooterManDev** [comment]
Unfortunately, I'm able to reproduce it, so I'm not sure why it's not working for you. The fix they provided didn't fix my issue, so I reported here. So I don't know what to do.

**Comment by MelanX** [comment]
I tried again and have no idea. Could you record a video of it, and upload it somewhere like YouTube? Maybe I find something out I'm doing wrong.

**Comment by FooterManDev** [comment]
Unfortunately I now cannot reproduce it for some reason 😕 I don't know what triggered the worldgen then.
* I tried in a new world, didn't work.
* I tried launching to Earth(overworld) from Moon 16 chunks away from original drop, didn't work.
* I tried launching to Earth(overworld) from all other planet dimensions.

I don't know what made it work before, and now I look like a fool. I apologize for the trouble, I'll try to figure it out tomorrow.
